lib/susedistribution.pm

```diff
@@ -201,7 +201,27 @@ sub init_consoles {
         $self->add_console('user-console',  'tty-console', {tty => 4});
         $self->add_console('x11',           'tty-console', {tty => 7});
     }
-    if (check_var('BACKEND', 'svirt') || check_var('BACKEND', 's390x')) {
+
+    # JeOS via svirt backend
+    if (get_var('JEOS') && check_var('BACKEND', 'svirt')) {
```

Comment:
we need to refactor these bits. the svirt backend is very generic, so you can't really rely on it, so you need extra variables that tell which kind of setup this is. JEOS is indeed a bit too specific, but it's the best we have atm